lightkurve/targetpixelfile.py

```diff
@@ -1336,7 +1336,9 @@ def __init__(self, path, quality_bitmask='default', **kwargs):
         # Early TESS releases had cadences with time=NaN (i.e. missing data)
         # which were not flagged by a QUALITY flag yet; the line below prevents
         # these cadences from being used. They would break most methods!
-        self.quality_mask &= np.isfinite(self.hdu[1].data['TIME'])
+        #self.quality_mask &= np.isfinite(self.hdu[1].data['TIME'])  # kjm2019JAN21: original
+        if (quality_bitmask > 0):  # kjm2019: new
+            self.quality_mask &= np.isfinite(self.hdu[1].data['TIME'])  # kjm2019JAN21: indented
```

Comment:
Please remove `# kjm2019JAN21: indented`, version control will keep track of time & author of changes!